src/middlewared/middlewared/api/v25_04_0/rdma.py

```diff
@@ -42,4 +42,4 @@ class RdmaCapableProtocolsArgs(BaseModel):
 
 
 class RdmaCapableProtocolsResult(BaseModel):
-    result: list[Literal["NFS"]]
+    result: list[Literal["iSER", "NFS"]]
```

Comment:
Not that I really care but there is (mostly) an unspoken "rule" to make these all upper-case.